src/main/java/duke/command/PrintCommand.java

```diff
@@ -0,0 +1,31 @@
+package duke.command;
+
+import duke.parts.Storage;
+import duke.parts.TaskList;
+import duke.parts.Ui;
+
+import java.io.IOException;
+
+/**
+ * Represents a command which is used to print the list of tasks.
+ * It is executed when the execute method is called.
+ */
+public class PrintCommand extends Command{
```

Comment:
For your functions, some of them have a whitespace at the brackets while some does not. I think with the checkstyle one it is preferred to have the whitespace